lib/Image/Image.js

```diff
@@ -11,24 +11,24 @@ var
 	Img = require('enyo/Image');
 
 var
-	ImageBadge = require('../ImageBadge');
+	Icon = require('../Icon'),
+	Overlay = require('../Overlay');
 
 /**
 * {@link moon.Image} is a simple control that wraps an {@link enyo.Image} to provide proper
 * alignment with text-based controls.
 *
-* In addition, `moon.Image` accepts optional {@link moon.ImageBadge} client components
-* (ImageBadge being the default kind of Image), which are placed inside a container positioned
-* over the image.  These badges are normally persistent, but may be shown or hidden based on
-* {@glossary Spotlight} focus, using the
-* [showBadgesOnSpotlight]{@link moon.Image#showBadgesOnSpotlight} property.
+* In addition, `moon.Image` adds {@link moon.Overlay} support to show controls over the image. This
+* can be used to add action icons as in the example below.
```

Comment:
It should be documented that `components` of this kind are automatically assumed to be `moon.Icon`s, unless an `overlayComponents` block exists. Also, if `overlayComponents` _do_ exist, they are **not** automatically treated as `moon.Icon`s.